tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -52,6 +57,22 @@
 EXTERNAL_GROUP_NAME = 'foobargroup'
 
 
+@pytest.fixture(scope='module')
+def subscribe_satellite():
+    """subscribe satellite to cdn"""
+    run_command(cmd='rpm -qa | grep katello-ca-consumer | xargs -r rpm -e')
+    os.environ['RHN_USERNAME'] = settings.subscription.rhn_username
```

Comment:
Why are we setting these environment variables on the system running python, when we need to subscribe the satellite?  Why use the ssh module to send one command, and fabric to send the others?

Reply:
subscribe in the function in automation tools repo, I thought why not use that which is tested many times.

Reply:
@mshriver there are some underneath subscription cleaning commands that will help while registering. Please let me know if this blocker to merge this PR.

Reply:
@mshriver as @lpramuk identified looks like my approach is invalid will update the codebase based on the run_command rather fabric